src/inc/utilcode.h

```diff
@@ -1490,6 +1490,15 @@ class CPUGroupInfo
 int GetCurrentProcessCpuCount();
 DWORD_PTR GetCurrentProcessCpuMask();
 
+size_t GetOsPageSize();
+
+#define PAGE_SIZE               GetOsPageSize()
```

Comment:
Windows build still expects this to be a constant.  I will put this in an #ifdef FEATURE_PAL block.